react-ui/src/components/Countdown.jsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * Time remaining to submit or vote
+ * Displayed in header
```

Comment:
It's also displayed in the entry description drawer component as well, since you can vote from there.